healthcheck/healthcheck.go

```diff
@@ -70,48 +81,56 @@ func (hc *HealthCheck) execute(ctx context.Context) {
 		currentBackend := backend
 		currentBackendID := backendID
 		safe.Go(func() {
+			hc.wg.Add(1)
+			defer hc.wg.Done()
+			log.Debugf("Initial healthcheck for currentBackend %s ", currentBackendID)
+			checkBackend(currentBackend)
+			ticker := time.NewTicker(currentBackend.Interval)
```

Comment:
Notice that I moved the ticker up. Didn't make sense to reinitialize the ticket inside the loop.